src/keymaster-api.js

```diff
@@ -462,7 +462,7 @@ v1router.delete('/issued/:did', async (req, res) => {
     }
 });
 
-v1router.get('/exportdid/:did/did', async (req, res) => {
+v1router.get('/exportdid/:did/export', async (req, res) => {
```

Comment:
I suggested `/dids/:did/export`. Oversight or disagreement?

Reply:
this was an oversight, updated now